src/utils.ts

```diff
@@ -140,35 +143,168 @@ export class Utility {
+        }
+    }
+
+    static getStationUpload(orgId: string) {
+        return {
+            tdei_org_id: orgId,
+            station_name: `${faker.animal.dog()} Station`,
+            polygon: this.getRandomPolygon()
+        }
+    }
+
+    static getServiceUpload(orgId: string) {
+        return {
+            tdei_org_id: orgId,
+            service_name: `${faker.animal.cat()} Service`,
+            polygon: this.getRandomPolygon()
+        }
+    }
+}
+
+export class APIUtility {
+    private instance: AxiosInstance;
+
+    constructor() {
+        axios.defaults.baseURL = config.seed.baseUrl;
+        this.instance = axios.create();
+        // console.log(Utility.getRandomOrganizationUpload())
+    }
+
+    async login(): Promise<void> {
+        try {
+            const resp = await axios({
+                method: 'post',
+                url: '/api/v1/authenticate',
+                data: {
+                    username: config.seed.adminUser,
+                    password: config.seed.adminPassword
+                }
+            })
+            const accessToken = resp?.data?.access_token
+            axios.defaults.headers.common['Authorization'] = `Bearer ${accessToken}`
+        } catch (err: any) {
+            throw err?.response?.data?.message
+        }
+    }
+
+    async createOrg(): Promise<string> {
+        try {
+            const resp = await axios({
+                method: 'post',
+                url: '/api/v1/organization',
+                data: Utility.getRandomOrganizationUpload()
+            })
+            return resp?.data?.data
+        } catch (err: any) {
+            throw err?.response?.data?.message
+        }
+    }
+
+    async createUser(): Promise<any> {
+        try {
+            const resp = await axios({
+                method: 'post',
+                url: '/api/v1/register',
+                data: Utility.getUserUpload()
+            })
+            return resp?.data?.data
+        } catch (err: any) {
+            throw err?.response?.data?.message
+        }
+    }
+
+    async addPermission(orgId: string, username: string, role: string): Promise<void> {
+        try {
+            const resp = await axios({
+                method: 'post',
+                url: '/api/v1/permission',
+                data: {
+                    tdei_org_id: orgId,
+                    user_name: username,
+                    roles: [role]
+                }
+            })
+            return resp?.data?.data
+        } catch (err: any) {
+            throw err?.response?.data?.message
+        }
+    }
+
+    async createStation(orgId: string): Promise<string> {
+        try {
+            const resp = await axios({
+                method: 'post',
+                url: '/api/v1/station',
+                data: Utility.getStationUpload(orgId)
+            })
+            return resp?.data?.data
+        } catch (err: any) {
+            throw err?.response?.data?.message
+        }
+    }
+
```

Comment:
remove Org or deactivate Org call is missing

Reply:
oohh! that i missed